invenio_app_ils/config.py

```diff
@@ -1294,12 +1294,24 @@ def _(x):
     "invenio_pidrelations.serializers.schemas.RelationSchema",
     "invenio_app_ils.records.relations.api.ParentChildRelation",
 )
+CHAIN_RELATION = ILS_RELATION_TYPE(
+    5,
+    "chain",
+    "Chain",
+    "invenio_app_ils.records.relations.nodes:PIDNodeRelated",
+    "invenio_pidrelations.serializers.schemas.RelationSchema",
+    "invenio_app_ils.records.relations.api.ChainRelation",
+)
+
 
 PARENT_CHILD_RELATION_TYPES = [MULTIPART_MONOGRAPH_RELATION, SERIAL_RELATION]
 
 SIBLINGS_RELATION_TYPES = [LANGUAGE_RELATION, EDITION_RELATION, OTHER_RELATION]
 
-ILS_PIDRELATIONS_TYPES = PARENT_CHILD_RELATION_TYPES + SIBLINGS_RELATION_TYPES
+LINKED_RELATION_TYPES = [CHAIN_RELATION]
```

Comment:
Nothing to do with your code, but I think we should not mix Chain and Link/Linked terms. I think we should find one and stick to it, WDYT? @kprzerwa @zzacharo any input too?
Checking the definition of [Linked list](https://en.wikipedia.org/wiki/Linked_list) for example, it is mentioned of order and sequence.

What about something like:
`SEQUENCES_RELATION_TYPES` = [SEQUENCE_RELATION]`

And then we stick everywhere to sequence or whatever term we chose.